packages/components/src/templates/next/components/internal/Navbar/NavItem.tsx

```diff
@@ -126,11 +126,11 @@ const Megamenu = ({
         showExternalIcon={isExternal}
         isWithFocusVisibleHighlight
         href={referenceLinkHref}
-        className="group inline-flex w-fit items-center gap-1.5 hover:text-brand-interaction-hover hover:no-underline"
+        className="group relative w-fit items-center gap-1.5 hover:text-brand-interaction-hover hover:no-underline"
       >
         {name}
         {!isExternal && (
-          <BiRightArrowAlt className="text-[1.5rem] transition ease-in group-hover:translate-x-2.5" />
+          <BiRightArrowAlt className="inline text-[1.5rem] transition ease-in group-hover:translate-x-2.5" />
         )}
```

Comment:
issue: hmm the arrow seems to be a bit misaligned from the centre of the text

<img width="437" height="270" alt="image" src="https://github.com/user-attachments/assets/c45b20e3-11d3-482a-a0e5-1c34e632f8a9" />